### ISSUE TITLE
sing-box geosite not found

### ISSUE THREAD

**Issue by Pakrohk** [MatsuriDayo/nekoray#389]
**Description of the problem**
Due to geosite.db problem, no proxy is activated on sing-box.
**Expected behavior:**
Activation of proxies
**Actual behavior:**
Show geosite.db not found warning
**How to reproduce**
add a vless or vmess proxy to the app and try to start it (aur repositories version)
![image](https://user-images.githubusercontent.com/27810360/219474716-0ce185ac-67a5-48c0-bf73-31ecff4f73e5.png)

**Comment by Pakrohk** [comment]
>  Did you install another aur package sing-geosite sing-geoip?

No, there are also other problems in the new version (I'm using plasma desktop), the right-click menu in the latte panel in the system tray doesn't work.
Nekobox and nekoray data storage is completely different.
If I click on the software launcher instead of the system tray, two icons will be displayed in the system tray at the same time.

**Comment by SepHeidari** [comment]
@Pakrohk  I solved it by manually placing the geosite.db and geoip.db inside /usr/lib/nekoray/

**Comment by Pakrohk** [comment]
> @Pakrohk I solved it by manually placing the geosite and geoip inside /usr/lib/nekoray/

Thank you, but I don't use nekoray for a long time and I use clash-verge instead, but it is better if this issue remains open, maybe it will cost someone.

**Comment by coco456** [comment]
同样遇到这个问题

**Comment by jieran233** [comment]
Make sure you have installed this packages first: `yay -S nekoray sing-geoip-common sing-geoip-db sing-geoip-rule-set sing-geosite-common sing-geosite-db sing-geosite-rule-set`

If still have the problem, try set `资源文件路径` as `/usr/share/sing-box`. It works for me

![图片](https://github.com/MatsuriDayo/nekoray/assets/63990576/dc8be197-15b6-408c-8d0a-f5dbd0b6b1b6)

**Comment by AA1999** [comment]
Packages `sing-geosite-common` and `sing-geoip-common` have been removed from the AUR and their git variants don't install without errors.

**Comment by linlinlin1992** [comment]
The dbs of the two files (geosite and geoip) in AUR cannot work with nekoray after the recent update. Downgraded the two dbs works.

**Comment by oulh** [comment]
The linux64.zip version I'm using doesn't have this error, maybe the appimage version is the same. The  [geoip.db](https://github.com/SagerNet/sing-geoip/releases)  and [geosite.db](https://github.com/SagerNet/sing-geosite/releases/) files can be updated manually.

**Comment by solitudealma** [comment]
> Make sure you have installed this packages first: `yay -S nekoray sing-geoip-common sing-geoip-db sing-geoip-rule-set sing-geosite-common sing-geosite-db sing-geosite-rule-set` from AUR or ArchLinuxCN Repo
> 
> If still have the problem, try set `资源文件路径` as `/usr/share/sing-box`. It works for me
> 
> ![图片](https://private-user-images.githubusercontent.com/63990576/316579032-dc8be197-15b6-408c-8d0a-f5dbd0b6b1b6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY2ODk4NDksIm5iZiI6MTcxNjY4OTU0OSwicGF0aCI6Ii82Mzk5MDU3Ni8zMTY1NzkwMzItZGM4YmUxOTctMTViNi00MDhjLThkMGEtZjVkYmQwYjZiMWI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI2VDAyMTIyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWIzY2FiOThiZmVkMzA4NGQ0ZDZmMDdlYzY0YTY4NThlNzRjM2NjMjI4YTBmN2YzZjRhZmIxYjdlYWFmZTY4ZWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0rDpILtf9O86-RVm-VXxrBqCcsCxpmb8htb4Tu-165E)

this is useful

**Comment by ludenticus** [comment]
> Make sure you have installed this packages first: `yay -S nekoray sing-geoip-common sing-geoip-db sing-geoip-rule-set sing-geosite-common sing-geosite-db sing-geosite-rule-set` from AUR or ArchLinuxCN Repo
> 
> If still have the problem, try set `资源文件路径` as `/usr/share/sing-box`. It works for me
> 
This is no longer valid in beta 4.0-beta3. You need to `ln -s /usr/share/sing-box /usr/share/sing-geoip`.

**Comment by javaj-r** [comment]
> > Make sure you have installed this packages first: `yay -S nekoray sing-geoip-common sing-geoip-db sing-geoip-rule-set sing-geosite-common sing-geosite-db sing-geosite-rule-set` from AUR or ArchLinuxCN Repo
> > If still have the problem, try set `资源文件路径` as `/usr/share/sing-box`. It works for me
> 
> This is no longer valid in beta 4.0-beta3. You need to `ln -s /usr/share/sing-box /usr/share/sing-geoip`.

sing-box directory setting is remove!

![Screenshot](https://github.com/user-attachments/assets/c64c8bb6-b0dc-42cd-bdc8-27c9384be477)

Thanks @ludenticus this worked for me `ln -s /usr/share/sing-box /usr/share/sing-geoip`.

**Comment by solitudealma** [comment]
> > > Make sure you have installed this packages first: `yay -S nekoray sing-geoip-common sing-geoip-db sing-geoip-rule-set sing-geosite-common sing-geosite-db sing-geosite-rule-set` from AUR or ArchLinuxCN Repo
> > > If still have the problem, try set `资源文件路径` as `/usr/share/sing-box`. It works for me
> > 
> > 
> > This is no longer valid in beta 4.0-beta3. You need to `ln -s /usr/share/sing-box /usr/share/sing-geoip`.
> 
> sing-box directory setting is remove!sing-box 目录设置已删除！
> 
> ![Screenshot](https://private-user-images.githubusercontent.com/34844437/349505809-c64c8bb6-b0dc-42cd-bdc8-27c9384be477.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyOTMyMjEsIm5iZiI6MTcyMzI5MjkyMSwicGF0aCI6Ii8zNDg0NDQzNy8zNDk1MDU4MDktYzY0YzhiYjYtYjBkYy00MmNkLWJkYzgtMjdjOTM4NGJlNDc3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODEwVDEyMjg0MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI5ZTMwYWFmOTJlYzUyMWI2ZGY0ODIzMzJkOWViMmY5Zjc5ZWExODM0NzFlYzBkZWU4NmFiN2U2ZGI2Nzg3MDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iwDz-lnOULVqf81OsKmnjGsgEjg4eCGwCG_c6BRhVyQ)
> 
> Thanks @ludenticus this worked for me `ln -s /usr/share/sing-box /usr/share/sing-geoip`.

still exists
